### DISCUSSION TITLE
How to correctly open notebook ?

### DISCUSSION THREAD

**Discussion by Alseidon** [fonsp/Pluto.jl#1970] [Questions about using Pluto] [answered]
So far I've had an excellent experience with Pluto, but something i found a bit tedious was opening the notebook at the beginning. All my notebooks are about six directories deep from the home directory (meaning it takes time to manually type the path), and if I move them from one directory to another, I can't use the "recently opened" feature (I think ?).

So I figured I would drag and drop them from the file explorer (where I have bookmarks) ; but the thing is, if I open foo.jl this way, the window will display a new notebook name (like Wonderful notes.jl), and any new modifications won't affect foo.jl.

So, does dropping notebooks work to open them ? Do you have a suggestion to speed things a bit ?

Thanks !

**Comment by disberd** [comment, marked as answer]
If you are on windows, you can right click the file in the explorer **while pressing shift** to find the *Copy as path* entry.
You can then paste that directly on the file picker bar inside Pluto.

If you are not on windows, I believe you might have some similar way to extract a file path from the file explorer

**Reply by Alseidon** [reply]
I am on Linux, but I can indeed copy the path this way - this should be enough for now. I wonder though why the drag and drop doesn't work...

**Reply by disberd** [reply]
I believe the file drop in html/javascript does not give you information on the path of the file you dropped, but just on its content. 
Probably that is the reason why dropping a file creates a new notebook with the dropped file contents rather than opening the original file.

**Reply by Alseidon** [reply]
Makes sense, thank you!
